source/include/jobs.h

```diff
@@ -33,14 +32,29 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/**
+ * @brief maximum length of a thing name for the AWS IoT Jobs Service
```

Comment:
Can you please capitalize the first letter of each first word and end the sentences with periods?

Reply:
changed here and elsewhere